app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -391,6 +391,7 @@ export default function () {
       function () {
         this.route("index", { path: "/" });
         this.route("show", { path: "/:plugin_id" }, function () {
+          this.route("schema", { path: "schema/:setting_name" });
```

Comment:
I'm not sure this should be under `/plugins`? This new feature is "type:object support for site settings", which means it applies to core settings as well as plugins?